### ISSUE TITLE
Library As Unique Repo

### ISSUE THREAD

**Issue by davidbitton** [alexmos/sbgc-api-examples#1]
Can you move the library to it's own repo so I can include it as a submodule in another repo? Thanks!

**Comment by alexmos** [comment]
Hi,  
Could you describe a structure how it may organized?  I have no big experience with the github so may do it wrong.

**Comment by binloan** [comment]
I guess @davidbitton wants that the "libraries" folder is maintained on the root of this repo and the examples are packed inside an "examples" folder. Then you can directly clone it e.g. in your "Arduino/libraries" folder and still have the examples together with the library.

**Comment by davidbitton** [comment]
No. The AlexMos serial library in use should be the one from their repo and it should be in this repo as a Git submodule.

**Comment by davidbitton** [comment]
@binloan yes on the clone, but I was thinking in reference the the PX4 codebase and using this as a Git submodule.
